src/unix/getaddrinfo.c

```diff
@@ -135,6 +158,24 @@ static void uv__getaddrinfo_done(struct uv__work* w, int status) {
 }
 
 
+static void uv__getaddrinfo_work(struct uv__work* w) {
+  uv_getaddrinfo_t* req;
+  int err;
+
+  req = container_of(w, uv_getaddrinfo_t, work_req);
```

Comment:
@cabelitos like this one (but different member, since this is the `uv__work` here, there it's the `uv__io_t` there)